src/chrome/content/scaffold/templates/newWeb.js

```diff
@@ -0,0 +1,75 @@
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2017 YOUR_NAME <- TODO
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+// attr()/text() v2
+function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}
+
+
+function detectWeb(doc, url) {
+	//TODO: adjust the logic here
+	if (url.includes('/article/')) {
+		return "newspaperArticle";
+	} else if (getSearchResults(doc, true)) {
+		return "multiple";
+	}
+}
+
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	//TODO: adjust the xpath
+	var rows = doc.querySelectorAll('h2>a.title[href*="/article/"]');
+	for (var i=0; i<rows.length; i++) {
+		//TODO: check and maybe adjust
```

Comment:
Space after `//` everywhere (but since these are templates, I'll also go over them in a separate commit and adjust some style things).